content/LCDC.md

```diff
@@ -49,11 +49,13 @@ one.
 
 #### LCDC.5 - Window enable
 
-This bit controls whether the window shall be displayed or not. (TODO:
-what happens when toggling this mid-scanline ?) This bit is overridden
-on DMG by [bit 0](#lcdc-0-bg-and-window-enable-priority)
+This bit controls whether the window shall be displayed or not.
+This bit is overridden on DMG by [bit 0](#lcdc-0-bg-and-window-enable-priority)
 if that bit is reset.
 
+Behavior of this register when changing it mid-frame is a bit complicated;
+please [see further below](#ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-7-r-w).
+
 Note that on CGB models, setting this bit to 0 then back to 1 mid-frame
```

Comment:
Should't this be reported in #ff4a-wy-window-y-position-r-w-ff4b-wx-window-x-position-7-r-w anyway?

Reply:
Sorry, wdym?